src/main/java/edu/ucsd/sbrg/escher/EscherConverter.java

```diff
@@ -228,6 +228,10 @@ public static EscherMap parseEscherJson(InputStream stream)
     // An Escher array contains meta-info about the map as the first object and the actual map as
     // second map.
     JsonNode escherJson = objectMapper.readTree(stream);
+    if(escherJson.get(0) == null) {
+      logger.severe(format(bundle.getString("EscherConverter.missingMetaInfo")));
+      throw new IOException("Missing or misplaced meta-info: File is malformatted"){};
```

Comment:
It would be preferable to use the message `String` from the bundle also for the `IOException`. Creating a variable of type `String` first to load the message could help.